models.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
```

Comment:
why do you need those weird names? You anyway don't import them directly, but using `models` prefix)